documentation/quarkus/guide-exception-handling.asciidoc

```diff
@@ -0,0 +1,65 @@
+:toc:  macro
+toc::[]
+
+= Exception Handling in Quarkus
+
+For link:../guide-exceptions.asciidoc[handling exceptions] within a Spring application, devon4j provides the https://github.com/devonfw/devon4j/tree/master/modules/rest[devon4j-rest module], which provides a https://github.com/devonfw/devon4j/blob/develop/modules/rest/src/main/java/com/devonfw/module/rest/service/impl/RestServiceExceptionFacade.java[`RestServiceExceptionFacade`] to handle all exceptions in a consistent way. Since the module is not suitable for Quarkus, we need to implement this ourselves.
+
+This guide shows how to do that. For an example, see also our https://github.com/devonfw-sample/devon4quarkus-reference[Quarkus reference application].
```

Comment:
"This guide shows how to do that" - sounds awkward. 
"This guide shows how to do just that" sounds better.

"For an example, see our [Quarkus reference application]" - **also** is not allowed here